runtime/test/mock-slot-composer.js

```diff
@@ -22,7 +22,11 @@ class MockSlot extends Slot {
     this._content = {};
   }
   setContent(content, handler) {
-    this._content = Object.assign(this._content, content);
+    if (content.template) {
```

Comment:
If `content.template` is falsy then this will not update the mock slot's `this._content`, but it will always update the slot's model from `content.model`. This sort of surprised me, but I assume it's because one might want to update the model without changing the template and having to pass the same template every time seems unnecessary?

Perhaps worth a comment on `setContent` to document.

Reply:
added comment (yes, your understanding is exactly correct - we don't pass the template every time, and i want the content in the mock slot be somewhat representative of what's actually rendered, i'll add more verifications for the slots content later).